app.ts

```diff
@@ -0,0 +1,226 @@
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		// Trigger async function
+		async function getColumnNamesCall(): Promise<string[]> {
+			const response = await fetch('http://localhost:2050/columns');
+			// let promise = new Promise((res, rej) => {
+			// 	setTimeout(() => res("Now it's done!"), 50)
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		// Trigger async function
+		async function getRecordsCall(fromID: number, toID: number): Promise<string[][]> {
+			const response = await fetch(`http://localhost:2050/records?from=${(fromID)}&to=${(toID)}`);
+			// let promise = new Promise((res, rej) => {
+			// 	setTimeout(() => res("Now it's done!"), 50)
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		//region Data Loading methods
+		async function LoadRecordsData(fromID: number, toID: number): Promise<number[]> {
+			const recordsvalue = await getRecordsCall(fromID, toID);
+			let DisplayContent = '';
+			for (const record of recordsvalue) {
+				DisplayContent += `<tr id="table-row-${record[0]}">`;
+				for (const column of record) {
+					DisplayContent += `<td align="center">${column}</td>`;
+				}
+				DisplayContent += '</tr>';
+				$("#wrapper-table-content-body").empty();
+				$("#wrapper-table-content-body").append(DisplayContent);
+				throw new Error("Error No data");
+			}
+			return [fromID, toID];
+		}
+
+		// Load Records Function
+		function RecordsFromCursor(cursor: number[]): Promise<number[]> {
+			cursor = cursor.sort((a, b) => { return a - b });
+			return LoadRecordsData(cursor[0], cursor[1]);
+			// throw new Error("Error");
+		}
+
+		// Handlers
+		async function LoadPageContent(fromID: number, toID: number): Promise<number[]> {
+			let DisplayContent = "";
+			const columns = await getColumnNamesCall();
+			for (const column of columns) {
+				DisplayContent += `<th align="center">${column}</th>`;
+				$("#wrapper-table-header-row").empty();
+				$("#wrapper-table-header-row").append(DisplayContent);
+			}
+			return LoadRecordsData(fromID, toID);
+		}
+
+		// Conversion Function
+		function ConvertNumber(input: string | number, parseAsInt: boolean = true): number {
+			switch (typeof input) {
+				case ('string'):
+					if (parseAsInt == true) {
+						return parseInt(input as string);
+					}
+					return parseFloat(input as string);
+				case ("number"):
+					return input as number;
+			}
+			return 0;
+		}
+
+		// Height Diplay Function
+		function calculateToId(fromId: number): number {
+			const possibleRecordsData = Math.max((window.innerHeight - ($("#form-content").innerHeight() as number)));
```

Comment:
Why are you getting the max of one number? And the way that you use the result does not make a lot of sens, this looks like it is adding the pixels to the fromId, as if a row is only one pixel tall. I think you accidentally removed a divide somewhere here.